### PR TITLE
[14.0] overcapacity & Log flooding fix

### PR DESCRIPTION
**Pull request by PieterPaulussen**

### CODE REVIEW

File: queue_job/jobrunner/channels.py [hunk 1]
```diff
@@ -477,7 +477,7 @@ def set_done(self, job):
         This removes it from the channel queue.
         """
         self.remove(job)
-        _logger.debug("job %s marked done in channel %s", job.uuid, self)
+        _logger.info("job %s marked done in channel %s", job.uuid, self)
```
Comment (PieterPaulussen) [inline queue_job/jobrunner/channels.py:480]:
```suggestion
       _logger.info("job %s test for peter %s", job.uuid, self)
```

**Comment by PieterPaulussen** [comment]
@StefanRijnhart @rconjour I finally got the overcapacity to play nicely with the new additions to the queue module.
Debugging was particularly difficult since it's very hard to guess during the debugging which test is actually performed at that time.
The best improvement to this module would be to replace those doctests with actual UnitTests.

**Review by rconjour — APPROVED** [review]

File: queue_job/jobrunner/channels.py [hunk 2]
```diff
@@ -692,10 +748,11 @@ class ChannelManager(object):
     >>> cm.get_wakeup_time()
     104
 
-    Let's test throttling in combination with a queue reaching full capacity.
+    Let's test throttling in combination with a queue reaching full capacity
+    (but without the root channel having more capacity available)
 
     >>> cm = ChannelManager()
-    >>> cm.simple_configure('root:4,T:2:throttle=2')
+    >>> cm.simple_configure('root:2,T:2:throttle=2')
```
Comment (StefanRijnhart) [inline queue_job/jobrunner/channels.py]: This change (as well as the change in the test below) looks suspicious. Does the combination of this 'native' throttling need special care in combination with overcapacity?

Comment (PieterPaulussen) [reply queue_job/jobrunner/channels.py]: I've adapted the code to ignore throttled channels. And restored the original throttling tests!

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**